data/editor.config

```diff
@@ -354,6 +354,7 @@
       <type id="amenity-restaurant" group="food">
         <include group="poi_internet" />
         <include field="cuisine" />
+        <include field="website_menu" />
```

Comment:
Please review all related POIs at least in group=food, and also add missing cuisine tags if necessary.

Why was it displayed for Parking?

Reply:
I forgot to add the field in details pane

Reply:
I think it's done. Also, I don't see missing cuisine tags.